src/collectd.conf.pod

```diff
@@ -3322,6 +3322,28 @@ TCP-Port to connect to. Defaults to B<411>.
 
 =back
 
+=head2 Plugin C<mcelog>
+
+The C<mcelog plugin> uses mcelog to retrieve machine check exceptions, etc.
```

Comment:
What does the 'etc.' stand for? It doesn't add much information, I'd either leave it out or extend the sentence.